code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1118,15 +1118,40 @@
 
 /datum/reagent/carbondioxide/reaction_obj(obj/O, reac_volume)
 	if((!O) || (!reac_volume))
-		return 0
+		return FALSE
 	var/temp = holder ? holder.chem_temp : T20C
 	O.atmos_spawn_air("co2=[reac_volume/5];TEMP=[temp]")
 
 /datum/reagent/carbondioxide/reaction_turf(turf/open/T, reac_volume)
 	if(istype(T))
 		var/temp = holder ? holder.chem_temp : T20C
 		T.atmos_spawn_air("co2=[reac_volume/5];TEMP=[temp]")
-	return
+
+/datum/reagent/decoxygen
+	name = "Decoxygen"
+	reagent_state = GAS
+	description = "An highly oxygen dense molecule which rapidly expands liberating oxygen if the pressure and chemical temperature is low enough."
+	color = "#5EECFF" // rgb : 94, 236, 255
+	taste_description = "oxygen"
+
+/datum/reagent/decoxygen/on_mob_life(mob/living/carbon/M)
+	M.Dizzy(1)
+	M.adjustOxyLoss(-1)
+	M.adjustStaminaLoss(-2)
+	..()
```

Comment:
Add false for updating_health on adjustOxyLoss and adjustStaminaLoss to avoid two calls to update health, instead return true from the proc.

i.e. (sorry no suggestions for multiline comments)
```
/datum/reagent/decoxygen/on_mob_life(mob/living/carbon/M)
	M.Dizzy(1)
	M.adjustOxyLoss(-1, FALSE)
	M.adjustStaminaLoss(-2, FALSE)
	..()
	. = 1
```